src/server/config.c

```
@@ -162,6 +162,19 @@ static void process_game_section(toml_table_t *root, obc_server_cfg_t *cfg)
     value = toml_table_bool(game, "friendly_fire");
     if (value.ok) cfg->friendly_fire = value.u.b;
 
+    value = toml_table_string(game, "friendly_fire_mode");
+    if (value.ok) {
+        str_copy(cfg->friendly_fire_mode, sizeof(cfg->friendly_fire_mode),
+                 value.u.s);
+        free(value.u.s);
+    }
+
+    value = toml_table_double(game, "friendly_fire_tolerance");
+    if (value.ok) cfg->friendly_fire_tolerance = value.u.d;
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)